master/getting-started/docker/index.md

```diff
@@ -1,4 +1,35 @@
 ---
 title: Calico with Docker
 ---
-More information coming soon!
+
+Calico drives container connectivity and implements network policy for Docker containers.
+
+Use the navigation bar on the left to view information on Calico for Docker,
+or continue reading for an overview of recommended guides to get started.
+
+
+## Installation
+
+#### [Prerequisites](installation/prerequisites)
+
+Information on running etcd and configuring Docker for multi-host networking.
+
+#### [Manual Install  Guide]({{site.baseurl}}/{{page.version}}/getting-started/docker/installation/manual)
+
+Learn the two-step process for launching Calico for Docker.
+
+## Quickstart with "Calico-Ready" Clusters
+
+#### [Vagrant CoreOS](installation/vagrant-coreos) 
+
+Quickly launch a local CoreOS cluster ready for Calico.
```

Comment:
Also I think the sentence should be reworded so that it's more like "Follow this guide to blah blah blah" or "This guide describes", otherwise the sentence seems a little out of place.